tools/lint/lint.py

```diff
@@ -665,6 +666,8 @@ def check_script_metadata(repo_root, path, f):
                 pass
             elif key == b"variant":
                 pass
+            elif key == b"quic":
```

Comment:
I feel like at some point this should be 
```
elif key not in (b"title", b"script", b"variant", b"quic"):
     errors.append(rules.UnknownMetadata.error(path, line_no=idx + 1))
```

Reply:
Let's make today the day. :)